src/components/Hero/_Hero.scss

```diff
@@ -67,6 +67,39 @@
   }
 }
 
+.hero--tertiary {
```

Comment:
For consitency, move the CSS for `.hero--tertiary` below the CSS for `.hero--secondary`.